core/src/main/java/org/mobilitydata/gtfsvalidator/table/GtfsTableDescriptor.java

```diff
@@ -12,6 +12,9 @@ public abstract class GtfsTableDescriptor<T extends GtfsEntity> {
   // True if the specified file is required in a feed.
   private boolean required;
 
+  // True if the specified file is conditionally required.
+  private boolean conditionallyRequired;
```

Comment:
We already have an annotation to _mark_ the conditional required files, [ConditionalRequired](https://github.com/MobilityData/gtfs-validator/blob/master/model/src/main/java/org/mobilitydata/gtfsvalidator/annotation/ConditionallyRequired.java)

Reply:
It is so much better! That was my initial thought, but failed to find the annotation...